packages/trace-viewer/src/index.tsx

```diff
@@ -39,5 +39,26 @@ import { WorkbenchLoader } from './ui/workbenchLoader';
     setInterval(function() { fetch('ping'); }, 10000);
   }
 
+  if (window.parent !== window && new URL(window.location.href).searchParams.has('embedded')) {
```

Comment:
Unfortunatelly no, because this code must run inside the iframe. But maybe create an `embedded.html` / `embedded.tsx`, similar to `index.tsx` and `uiMode.tsx`, we can just drop this code there as well as instanciating the parameterized `WorkbenchLoader`

Reply:
> Unfortunatelly no, because this code must run inside the iframe.

I understand that the snapshot renderer should run in an iframe, but the app itself does not have to. Are you doing this for convenience?

Reply:
The app depends on the SW, and for that reason there's no easy way to render it without the iframe because [webviews don't support SW ](https://github.com/microsoft/vscode/issues/194751)